packages/trader/src/App/Components/Elements/PositionsDrawer/helpers/positions-helper.ts

```diff
@@ -1,16 +1,19 @@
 import { localize } from '@deriv/translations';
-import { isHighLow, getContractTypesConfig, isCallPut, isVanillaContract } from '@deriv/shared';
+import { isHighLow, getContractTypesConfig, isCallPut, isVanillaContract, TContractInfo } from '@deriv/shared';
 
-export const addCommaToNumber = (num, decimal_places) => {
-    if (!num || isNaN(num)) {
+export const addCommaToNumber = (
+    num: number | null | undefined,
+    decimal_places?: number | undefined
+): string | number | null | undefined => {
+    if (!num || isNaN(+num)) {
```

Comment:
If `num === 0`, then `!num === true` and we `return num`; Just to clarify, there is no need in adding comma to zero, correct? So we should return 0 as a result of function

Reply:
you mean this expression?

```
if (!num || isNaN(+num)) {
        return num;
    }
```
first part of if is to handle when num is 0, null or undefined.  so you just return what was initially passed in because you wont add a comma to 0, null or undefined.

second part of if is to handle when num is not a number (string, object, boolean, array, etc), it just returns whatever its passed in because we don't want to add a comma to those data types.

We only want to add comma to number as function name suggest so we early return `data(num)` that is passed in if type is not a number.